integration-tests/azure-cosmos/src/main/java/io/quarkiverse/azure/cosmos/it/CosmosResource.java

```diff
@@ -0,0 +1,82 @@
+package io.quarkiverse.azure.cosmos.it;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.*;
+
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.models.*;
+
+@Path("/quarkus-azure-cosmos")
+@ApplicationScoped
+public class CosmosResource {
+
+    @Inject
+    CosmosClient cosmosClient;
+
+    @Path("/{database}/{container}")
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response createItem(
+            @PathParam("database") String database,
+            @PathParam("container") String container,
+            Item body,
+            @Context UriInfo uriInfo) {
+
+        getContainer(database, container).upsertItem(body);
```

Comment:
Is it possible that the post method parameters are invalid, like `database` is empty.

Reply:
Do we need do some validation work here?

Reply:
The code is developed for integration-test / code sample of `azure-cosmos` extension, instead of part of extension we released. From this standpoint of view, only happy path on how to utilize the clients exposed by the extension is implemented, which should be reasonable.